pkg/util/proxyutil/reverse_proxy_test.go

```diff
@@ -180,7 +180,8 @@ func TestReverseProxy(t *testing.T) {
 			{status: 599, expectedSource: requestmeta.StatusSourceDownstream},
 		}
 
-		for _, tc := range testCases {
+		for _, testCase := range testCases {
+			tc := testCase
```

Comment:
out of curiosity: why we need this change?

Reply:
To resolve a flaky test happening in grafana enterprise